src/common/helper.js

```diff
@@ -2287,6 +2190,31 @@ function verifyZoomLinkToken (token) {
   }
 }
 
+/**
+ * Check if the user id (Number) exists in member-api
+ * If it exists return it, otherwise throw en error
+ * @param {number} userId - The Topcoder userId to check
+ */
+async function ensureTopcoderUserIdExists (userId) {
```

Comment:
Any concerns with API throttling if this is called 200 times in a short period of time?

Reply:
Actually, it doesn't appear to be used in the script, so you can probably disregard my question here.